scripts/ci/npm/publish-libs.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR=${DIR}/../../..
+DIST_DIR=${ROOT_DIR}/dist/@alfresco
+PROJECTS_DIR=${ROOT_DIR}/projects
+
+for var in "$@"
+do
+    case "$var" in
+      --dryrun) RUN_DRY=true
+      echo -e "Run with --dryrun, therfore script won't publish packages\n" ;;
```

Comment:
should be "therefore" ?

Reply:
yep, fixed, lesson learned, `cspell` is not working for `sh`